.github/workflows/smoke_tests.yml

```diff
@@ -0,0 +1,150 @@
+
+  git_dev_version:
+    name: Git Dev Version - ${{ matrix.compression }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 25
+
+    needs: pre_job
+    # NOTE: We always want to run job on master branch
+    if: ${{ needs.pre_job.outputs.should_skip != 'true' || github.ref == 'refs/heads/master' }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        compression:
+          - "deflate"
+          - "zstandard"
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Python 3.8
+        uses: actions/setup-python@v2
+        id: setup-python
+        with:
+          python-version: 3.8
+
+      - name: Setup jRuby
+        uses: ruby/setup-ruby@8029ebd6e5bd8f4e0d6f7623ea76a01ec5b1010d  # v1.110.0
+        with:
+          ruby-version: 'jruby'
+          bundler-cache: true
+
+      - name: Build Gem
+        run: |
+          gem build logstash-output-scalyr.gemspec
+          # We rename the file so we don't need to hard code version in Dockerfile
+          mv logstash-output-scalyr-*.gem logstash-output-scalyr.gem
+
+      - name: Restore Docker Layer Cache
+        uses: satackey/action-docker-layer-caching@46d2c640b1d8ef50d185452ad6fb324e6bd1d052  # v0.0.11
+        # Ignore the failure of a step and avoid terminating the job.
+        continue-on-error: true
+
+      - name: Run Smoke Tests
+        env:
+          SCALYR_SERVER: ${{ secrets.SCALYR_SERVER }}
+          SCALYR_API_KEY: ${{ secrets.SCALYR_API_KEY }}
+          READ_API_KEY: ${{ secrets.READ_API_KEY }}
+        run: |
+          gemfile=$(ls *.gem)
+          echo "Using gemfile: ${gemfile}"
+          cp $gemfile .circleci/docker/
+          # NOTE: We need script workaround due to no tty - https://github.com/actions/runner/issues/808
+          script -e -c ".circleci/smoketest_logstash.sh yarnsh/agent-testing:latest 300 $gemfile .circleci/docker 1"
+
+  git_stable_version:
+    name: Stable / Beta RubyGem Version - ${{ matrix.compression }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 25
+
+    needs: pre_job
+    # NOTE: We always want to run job on master branch
+    if: ${{ needs.pre_job.outputs.should_skip != 'true' || github.ref == 'refs/heads/master' }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        compression:
+          - "deflate"
+          - "zstandard"
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Python 3.8
+        uses: actions/setup-python@v2
+        id: setup-python
+        with:
+          python-version: 3.8
+
+      - name: Setup jRuby
+        uses: ruby/setup-ruby@8029ebd6e5bd8f4e0d6f7623ea76a01ec5b1010d  # v1.110.0
+        with:
+          ruby-version: 'jruby'
+          bundler-cache: true
+
+      - name: Download Gem From RubyGems
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y jq
```

Comment:
FYI jq is already installed on the GH-provided runners: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md#tools

Reply:
Cool, thanks. Will remove this + update step, should save us a couple of seconds :)